disasterinfosite/admin.py

```diff
@@ -7,7 +7,7 @@
 # GENERATED CODE GOES HERE
 # DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
 # adminModelImports
-from .models import EmbedSnugget, TextSnugget, SnuggetSection, Location, SiteSettings, SupplyKit, ImportantLink
+from .models import EmbedSnugget, TextSnugget, SnuggetSection, Location, SiteSettings, SupplyKit, ImportantLink, RDPOLiquefact_Clark, RDPO_counties, RDPOLiquefaction_OR, RDPOCascadiaM9_3_Clark, RDPO_region, RDPOCascadiaM9_OR
```

Comment:
Can we leave the stuff that's specific to the sample data out of these files (e.g. have something checked into the repo that is as if we have never run `import.py`)?

Reply:
I'm glad to have the sample data in there but I think it'll be easier to create new projects from this repo if we don't have the generated code stuff in here.

Reply:
Oooooohhhhhh, I'm out of practice with this project....  I'll fix that this afternoon and make a fresh commit.

Reply:
I know that feeling!